packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -792,12 +827,27 @@ export class SankeyChartBase extends React.Component<ISankeyChartProps, ISankeyC
           ref={(rootElem: HTMLDivElement) => (this.chartContainer = rootElem)}
           onMouseLeave={this._onCloseCallout}
         >
-          <FocusZone direction={FocusZoneDirection.bidirectional}>
+          <FocusZone direction={FocusZoneDirection.vertical}>
             <svg width={width} height={height} id={this._chartId}>
-              <g className={classNames.links} strokeOpacity={1}>
-                {linkData}
-              </g>
-              <g className={classNames.nodes}>{nodeData}</g>
+              {nodeLinkDomOrderArray.map(item => {
+                if (item.type === 'node') {
+                  return (
+                    <g key={nodes[item.index].nodeId} className={classNames.nodes}>
+                      {nodeData![item.index]}
+                    </g>
+                  );
+                }
+
+                return (
+                  <g
+                    key={`${(links[item.index].source as SNode).nodeId}-${(links[item.index].target as SNode).nodeId}`}
+                    className={classNames.links}
+                    strokeOpacity={1}
+                  >
+                    {linkData![item.index]}
+                  </g>
+                );
```

Comment:
Add the logic details that you have mentioned in the PR description as a comment in the code also.

Reply:
done